integrationExamples/gpt/userId_example.html

```diff
@@ -283,6 +283,12 @@
                         },
                           {
                             name: "uid2"
+                          },
+                          {
+                            name: "originFloc",
+                            params: {
+                              token: "SHARED_ID_TOKEN"
```

Comment:
I can't find the value of this token in your PR yet you say it is the default.

Reply:
@patmmccann For token, should the publisher register one based on their domain? For testing this module, I used the chrome version 91.0.4469.4 (Official Build) dev (x86_64) and enabled --enable-blink-features=InterestCohortAPI 
--enable-features="FederatedLearningOfCohorts:update_interval/10s/minimum_history_domain_size_required/1,FlocIdSortingLshBasedComputation,InterestCohortFeaturePolicy"

With these flags enabled, the token is not required and also for checking if "interest-cohort" is available, the chrome flag "Experimental Web Platform features" had to be enabled.

Reply:
I was also a bit confused like @patmmccann - in all the examples it says to use a Share ID token but doesn't mention what it is. If you don't need a token, then I think you should include a description for how to operate without one so it's clear for publishers.

I would suggest either: 
1. change the text in the examples to be something like `"INSERT_PUBLISHER_TOKEN"` and then provide links or a description of how to obtain it; OR
2. actually provide a valid token to use as a default

Reply:
@SKOCHERI @smenzer My understanding is that for the origin trial, browsers that are not explicitly setting flags (~5% of non-EEA Chrome instances) will need a token set before they will respond with a cohort ID. Setting flags overrides this behavior. 

To avoid publishers _having_ to enroll in the Origin trial process, I'd recommend setting a default token. I just obtained `A3dHTSoNUMjjERBLlrvJSelNnwWUCwVQhZ5tNQ+sll7y+LkPPVZXtB77u2y7CweRIxiYaGwGXNlW1/dFp8VMEgIAAAB+eyJvcmlnaW4iOiJodHRwczovL3NoYXJlZGlkLm9yZzo0NDMiLCJmZWF0dXJlIjoiSW50ZXJlc3RDb2hvcnRBUEkiLCJleHBpcnkiOjE2MjYyMjA3OTksImlzU3ViZG9tYWluIjp0cnVlLCJpc1RoaXJkUGFydHkiOnRydWV9` for sharedid.org origin. Can we set that as the fallback value?